### PR TITLE
V0.7.7

### PR DESCRIPTION
**Pull request by jgoppert**

### CODE REVIEW

**Comment by jakirkham** [comment]
The issue is `conda-build` now deletes the source directory before entering tests. While it never guaranteed that the source directory would exist, people were able to get away with it. However, this won't work now. Meaning this [line]( https://github.com/conda-forge/px4tools-feedstock/blob/272d4bc39c66641a7547653652b7711e0b47d8ec/recipe/meta.yaml#L36 ) is invalid and the cause of the problem. Instead one must tell `conda-build` to move some or all source files using [this]( http://conda.pydata.org/docs/building/meta-yaml.html#source-files ).

**Comment by jgoppert** [comment]
@jakirkham Thanks I had been stuck awhile, really helped.

**Comment by jakirkham** [comment]
No problem. 😀

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,3 +44,5 @@ about:
 extra:
   recipe-maintainers:
     - jgoppert
+
+#  vim: set et fenc=utf-8 ff=unix sts=0 sw=2 ts=2 :
```
Comment (jakirkham) [inline recipe/meta.yaml:48]: Guessing this was unintentional.

**Pull request merged**